File: Program's_Contributed_By_Contributors/Python_Programs/Sorting/selectionSort.py
array = [14, 4 , 56, 2, 100, 97]
l = len(array)
for i in range(l-1):
    temp = i
    for j in range(i+1, l):
        if array[j] < array[temp]:
            temp = j
    array[i], array[temp] = array[temp], array[i]

print(array)

File: Program's_Contributed_By_Contributors/Java_Programs/Sorting_Algorithms/Selection_Sort.java

public class Selection_Sort {
    public int[] sortArray(int nums[]) {
        for (int i = 0; i < nums.length; i++) {
            int index = i;
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[index] > nums[j]) {
                    index = j;
                }
            }
            int temp = nums[index];
            nums[index] = nums[i];
            nums[i] = temp;
        }
        return nums;
    }
}
